Specs/DateTime/Japanese/DatePeriodExtractor.json

```diff
@@ -2212,22 +2062,28 @@
   },
   {
     "Input": "１５００は年を示そうです。",
-    "NotSupported": "dotnet, javascript, python, java",
+    "NotSupported": "javascript, python, java",
     "Results": [
       {
         "Text": "1500",
         "Type": "daterange",
         "Start": 0,
         "Length": 4
+      },
+      {
+        "Text": "年",
```

Comment:
Shouldn't year at most be a duration? Or supressed if just a "unit" for an actual year?

Reply:
yes, it had been kept because it can be equivalent to "the year" in English. But since it is problematic, we added an ambiguity filter to remove its extraction.